src/client/Microsoft.Identity.Client/Platforms/Shared/DefaultOSBrowser/HttpListenerInterceptor.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.Platforms.Shared.Desktop.OsBrowser;
+
+namespace Microsoft.Identity.Client.Platforms.Shared.DefaultOSBrowser
+{
+    internal class HttpListnerInterceptor : IUriInterceptor
```

Comment:
I think there is typo?
HttpListnerInterceptor => HttpList**e**nerInterceptor  ?

Reply:
@jmprieur ---wow...good catch.